panic: store not initialized [1.9.3-rc.1]

### Summary
<!--
Please provide a clear and concise description of what the bug is.
-->

`gopass-jsonapi` panics when being run with any parameter. Same happens `gopass-hibp`, maybe also the other binaries.

### Steps To Reproduce
<!--
Steps to reproduce the problem
-->

```
gopass-1.9.3-rc.1-darwin-amd64 ./gopass-jsonapi
panic: store not initialized. run gopass init first

goroutine 1 [running]:
main.main()
	/home/tex/src/go/src/github.com/gopasspw/gopass/cmd/gopass-jsonapi/main.go:39 +0x9da
```

### Expected behavior
<!--
A clear and concise description of what you expected to happen.
-->

Command works as expected.

### Environment
<!--
Please complete the following information (see note below)
-->

- OS: macOS Catalina
- OS version: [uname -a]
- gopass Version: gopass 1.9.3-rc.1+a39aeeb9bdc304526dd9af477ffa0713a727d446 (a39aeeb9bdc304526dd9af477ffa0713a727d446) go1.14.2 darwin amd64
- Installation method: gopass repo binary and from source

<!--
**PLEASE NOTE**

There is a package named gopass in the official Debian repository.
This package is not related to this project in any way. If you
installed gopass from the Debian archives report any bugs in
the Debian BTS.
-->

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -13,10 +13,14 @@ before:
 builds:
   - id: gopass
     binary: gopass
-    flags: |
-      -trimpath -tags netgo -gcflags="-trimpath=$GOPATH" -asmflags="-trimpath=$GOPATH"
+    flags:
+      - -trimpath -tags netgo
     env:
       - CGO_ENABLED=0
+    asmflags:
+      - all=-trimpath={{.Env.GOPATH}}
+    gcflags:
+      - all=-trimpath={{.Env.GOPATH}}
     ldflags: |
       -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.CommitDate}} -extldflags '-static'
     goos:
@@ -36,10 +40,14 @@ builds:
   - id: gopass-git-credentials
     dir: cmd/gopass-git-credentials
     binary: gopass-git-credentials
-    flags: |
-      -trimpath -tags netgo -gcflags="-trimpath=$GOPATH" -asmflags="-trimpath=$GOPATH"
+    flags:
+      - -trimpath -tags netgo
     env:
       - CGO_ENABLED=0
+    asmflags:
+      - all=-trimpath={{.Env.GOPATH}}
+    gcflags:
+      - all=-trimpath={{.Env.GOPATH}}
     ldflags: |
       -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.CommitDate}} -extldflags '-static'
     goos:
@@ -52,10 +60,14 @@ builds:
   - id: gopass-hibp
     dir: cmd/gopass-hibp
     binary: gopass-hibp
-    flags: |
-      -trimpath -tags netgo -gcflags="-trimpath=$GOPATH" -asmflags="-trimpath=$GOPATH"
+    flags:
+      - -trimpath
     env:
       - CGO_ENABLED=0
+    asmflags:
+      - all=-trimpath={{.Env.GOPATH}}
+    gcflags:
+      - all=-trimpath={{.Env.GOPATH}}
     ldflags: |
       -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.CommitDate}} -extldflags '-static'
     goos:
@@ -68,10 +80,14 @@ builds:
   - id: gopass-jsonapi
     dir: cmd/gopass-jsonapi
     binary: gopass-jsonapi
-    flags: |
-      -tags netgo -gcflags="-trimpath=$GOPATH" -asmflags="-trimpath=$GOPATH"
+    flags:
+      - -trimpath -tags netgo
     env:
       - CGO_ENABLED=0
+    asmflags:
+      - all=-trimpath={{.Env.GOPATH}}
+    gcflags:
+      - all=-trimpath={{.Env.GOPATH}}
     ldflags: |
       -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.CommitDate}} -extldflags '-static'
     goos:
@@ -84,10 +100,14 @@ builds:
   - id: gopass-summon-provider
     dir: cmd/gopass-summon-provider
     binary: gopass-summon-provider
-    flags: |
-      -trimpath -tags netgo -gcflags="-trimpath=$GOPATH" -asmflags="-trimpath=$GOPATH"
+    flags:
+      - -trimpath -tags netgo
     env:
       - CGO_ENABLED=0
+    asmflags:
+      - all=-trimpath={{.Env.GOPATH}}
+    gcflags:
+      - all=-trimpath={{.Env.GOPATH}}
     ldflags: |
       -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.CommitDate}} -extldflags '-static'
     goos:

--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@ FISH_COMPLETION_OUTPUT    := fish.completion
 ZSH_COMPLETION_OUTPUT     := zsh.completion
 # Support reproducible builds by embedding date according to SOURCE_DATE_EPOCH if present
 DATE                      := $(shell date -u -d "@$(SOURCE_DATE_EPOCH)" '+%FT%T%z' 2>/dev/null || date -u '+%FT%T%z')
-BUILDFLAGS_NOPIE                := -ldflags="-s -w -X main.version=$(GOPASS_VERSION) -X main.commit=$(GOPASS_REVISION) -X main.date=$(DATE)" -gcflags="-trimpath=$(GOPATH)" -asmflags="-trimpath=$(GOPATH)"
+BUILDFLAGS_NOPIE          := -trimpath -ldflags="-s -w -X main.version=$(GOPASS_VERSION) -X main.commit=$(GOPASS_REVISION) -X main.date=$(DATE)" -gcflags="-trimpath=$(GOPATH)" -asmflags="-trimpath=$(GOPATH)"
 BUILDFLAGS                ?= $(BUILDFLAGS_NOPIE) -buildmode=pie
 TESTFLAGS                 ?=
 PWD                       := $(shell pwd)

--- a/cmd/gopass-git-credentials/main.go
+++ b/cmd/gopass-git-credentials/main.go
@@ -6,6 +6,7 @@ import (
 	"os"
 	"os/signal"
 
+	"github.com/gopasspw/gopass/internal/out"
 	"github.com/gopasspw/gopass/pkg/ctxutil"
 	"github.com/gopasspw/gopass/pkg/gopass/api"
 	"github.com/urfave/cli/v2"
@@ -13,6 +14,11 @@ import (
 
 const (
 	name = "gopass-git-credentials"
+)
+
+var (
+	// Version is the released version of gopass
+	version string
 )
 
 func main() {
@@ -42,7 +48,8 @@ func main() {
 
 	gp, err := api.New(ctx)
 	if err != nil {
-		panic(err)
+		out.Red(ctx, "Failed to initialize gopass API: %s", err)
+		os.Exit(1)
 	}
 
 	gc := &gc{
@@ -51,7 +58,7 @@ func main() {
 
 	app := cli.NewApp()
 	app.Name = name
-	app.Version = "0.0.1"
+	app.Version = version
 	app.Usage = `Use "!gopass-git-credentials $@" as git's credential.helper`
 	app.Description = "" +
 		"This command allows you to cache your git-credentials with gopass." +

--- a/cmd/gopass-hibp/main.go
+++ b/cmd/gopass-hibp/main.go
@@ -6,12 +6,18 @@ import (
 	"os"
 	"os/signal"
 
+	"github.com/gopasspw/gopass/internal/out"
 	"github.com/gopasspw/gopass/pkg/gopass/api"
 	"github.com/urfave/cli/v2"
 )
 
 const (
 	name = "gopass-hibp"
+)
+
+var (
+	// Version is the released version of gopass
+	version string
 )
 
 func main() {
@@ -35,7 +41,8 @@ func main() {
 
 	gp, err := api.New(ctx)
 	if err != nil {
-		panic(err)
+		out.Red(ctx, "Failed to initialize gopass API: %s", err)
+		os.Exit(1)
 	}
 
 	hibp := &hibp{
@@ -44,7 +51,7 @@ func main() {
 
 	app := cli.NewApp()
 	app.Name = name
-	app.Version = "TODO"
+	app.Version = version
 	app.Usage = "haveibeenpwned.com leak checker for gopass"
 	app.EnableBashCompletion = true
 	app.Commands = []*cli.Command{

--- a/cmd/gopass-jsonapi/main.go
+++ b/cmd/gopass-jsonapi/main.go
@@ -7,12 +7,18 @@ import (
 	"os/signal"
 	"strings"
 
+	"github.com/gopasspw/gopass/internal/out"
 	"github.com/gopasspw/gopass/pkg/gopass/api"
 	"github.com/urfave/cli/v2"
 )
 
 const (
 	name = "gopass-jsonapi"
+)
+
+var (
+	// Version is the released version of gopass
+	version string
 )
 
 func main() {
@@ -36,7 +42,8 @@ func main() {
 
 	gp, err := api.New(ctx)
 	if err != nil {
-		panic(err)
+		out.Red(ctx, "Failed to initialize gopass API: %s", err)
+		os.Exit(1)
 	}
 
 	ja := &jsonapiCLI{
@@ -45,7 +52,7 @@ func main() {
 
 	app := cli.NewApp()
 	app.Name = name
-	app.Version = "TODO"
+	app.Version = version
 	app.Usage = "Setup and run gopass-jsonapi as native messaging hosts, e.g. for browser plugins"
 	app.EnableBashCompletion = true
 	app.Action = func(c *cli.Context) error {

--- a/cmd/gopass-summon-provider/main.go
+++ b/cmd/gopass-summon-provider/main.go
@@ -6,12 +6,18 @@ import (
 	"os"
 	"os/signal"
 
+	"github.com/gopasspw/gopass/internal/out"
 	"github.com/gopasspw/gopass/pkg/gopass/api"
 	"github.com/urfave/cli/v2"
 )
 
 const (
 	name = "summon-gopass"
+)
+
+var (
+	// Version is the released version of gopass
+	version string
 )
 
 func main() {
@@ -35,7 +41,8 @@ func main() {
 
 	gp, err := api.New(ctx)
 	if err != nil {
-		panic(err)
+		out.Red(ctx, "Failed to initialize gopass API: %s", err)
+		os.Exit(1)
 	}
 
 	gc := &gc{
@@ -44,7 +51,7 @@ func main() {
 
 	app := cli.NewApp()
 	app.Name = name
-	app.Version = "0.0.1"
+	app.Version = version
 	app.Usage = `Use "gopass-summon-provider" as provider for "summon"`
 	app.Description = "" +
 		"This command allows to use gopass as a secret provider for summon." +

